src/pages/General.vue

```diff
@@ -91,6 +103,15 @@ export default {
     updateTelemetry(value) {
       ipcRenderer.invoke('settings-write', { telemetry: value });
     },
+    onNetworkUpdate(status) {
+      this.networkStatus = status;
+      ipcRenderer.send('update-network-status', status);
```

Comment:
We're triggering the change off the window? Does that me if I go offline while a different tab is selected (or the window is closed), we don't capture the state change (the tray icon is out of date)?

Reply:
The event is processed if a different tab is selected, but not if the window is closed, so this isn't going to work.

Reply:
Shouldn't this code be removed then?

Reply:
No, because the code does work while the window is displayed.